bitrise.yml

```diff
@@ -251,7 +251,64 @@ workflows:
         stack: osx-xcode-12.5.x
         machine_type_id: g2.8core
 
-  L10nBuild:
+  L10nScreenshotsTests:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@4: {}
+    - cache-pull@2: {}
+    - certificate-and-profile-installer@1: {}
+    - script@1:
+        title: Set Default Web Browser Entitlement
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -x
+            /usr/libexec/PlistBuddy -c "Add :com.apple.developer.web-browser bool true" Focus.entitlements
+            /usr/libexec/PlistBuddy -c "Add :com.apple.developer.web-browser bool true" Klar.entitlements
+    - script@1:
+        inputs:
+        - content: >-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+            # debug log
+            set -x
+            curl --location --retry 5 --output l10n-screenshots-dd.zip
+            "$MOZ_DERIVED_DATA_PATH"
+            mkdir l10n-screenshots-dd
+            unzip l10n-screenshots-dd.zip -d l10n-screenshots-dd
+            rm l10n-screenshots-dd.zip
+        title: Download derived data path
+    - script@1:
+        title: Generate screenshots
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+            # debug log
+            set -x
+
+            ./screenshots.sh --test-without-building $MOZ_LOCALES
+
+            mkdir -p artifacts
+
+            for locale in $(echo $MOZ_LOCALES); do
+              for product in Focus Klar; do
+                zip -9 -j "$locale-$product.zip" "l10n-screenshots/$product/$locale/"*
```

Comment:
The idea would be to have the zip files name as:` fr-focus.zip` similarly as how they are now for firefox-iOS https://app.bitrise.io/build/e407df2adc2b1510#?tab=artifacts. 
We need to specify the file to store in artifacts, it does not work to point to a dir, only to specific files.